superbuild/CMakeLists.txt

```diff
@@ -556,27 +731,36 @@ else()
     set(CMAKE_C_COMPILER   cc)
     set(CMAKE_CXX_COMPILER c++)
 endif()
-]])
+]] @ONLY)
 
 # Base toolchain file for building ISPC and LLVM dependencies.
 file(CONFIGURE OUTPUT stage2-toolchain.cmake CONTENT [[
 @CCACHE_LAUNCHER_ACTION@(CMAKE_C_COMPILER_LAUNCHER   @CCACHE_EXE@)
 @CCACHE_LAUNCHER_ACTION@(CMAKE_CXX_COMPILER_LAUNCHER @CCACHE_EXE@)
-set(ENV{PATH} "@ENV_PATH_STAGE2@")
+set(ENV{PATH}       "@ENV_PATH_STAGE2@")
```

Comment:
Should `PATH` be appended to instead of set?

Reply:
Also is the additional whitespace accidental?

Reply:
It shouldn't because ENV_PATH_STAGE2 is full env not the extra path to append to. It was appended earlier here https://github.com/ispc/ispc/blob/fce2cc3599023ff0c9a5797d8d91f60f597c6b38/superbuild/CMakeLists.txt#L527